Resources/Private/Build/MediaConsentPlugin/src/mocks/handlers.ts

```diff
@@ -0,0 +1,17 @@
+import { rest } from 'msw';
+
+export const handlers = [
+    rest.get('/api/get-content', (_req, res, ctx) => {
+        const htmlContent = `
+          <div id="c304" class="frame frame-default frame-type-mediaconsent_cns frame-layout-0">
```

Comment:
The "id" is a specific tt_content record ID. Are you sure, you will get 304 on all installations?

Reply:
this is an example content wrap and this id is not used anywhere.